packages/plugin-commands-setup/src/setup.ts

```diff
@@ -106,3 +109,50 @@ set -gx PATH "$PNPM_HOME" $PATH
 `, 'utf8')
   return `Updated ${configFile}`
 }
+
+async function setupEnvironmentPath (pnpmHomeDir: string): Promise<string> {
```

Comment:
you can move this function to a separate file and run unit tests for it by mocking execa